docs/modules/ROOT/pages/documentation/ecosystem/intellij-plugin/README.adoc

```diff
@@ -0,0 +1,34 @@
+= Payara IntelliJ IDEA Plugin
+
+Payara Platform provides the Payara IntelliJ IDEA tools to integrate Payara Server and Payara Micro into IntelliJ.
+The extension is available in the https://plugins.jetbrains.com/plugin/15114-payara-tools[JetBrains Plugins Marketplace].
+
+* <<installing-plugin,Installing Payara IntelliJ plugin>>
+* <<tools,Payara Platform Tools>>
+
+[[installing-plugin]]
+== Installing Payara IntelliJ plugin
+
+1. You can install plugin from the main menu File ? Settings ? Plugins dialog.
+
+2. In the Plugins dialog, switch to the Marketplace tab.
+
+3. In the dialog that opens, search for *Payara Tools*.
+
+4. Click the *Install* button:
+image::intellij-plugin/install-marketplace.png[alt=Visual Studio Marketplace, width="60%"]
+
+5. Click *OK* and apply the changes, and restart IntelliJ IDEA if prompted.
+
+[[tools]]
+== Payara Platform Tools
+
+* xref:documentation/ecosystem/intellij-plugin/payara-server.adoc[Payara Server] - 
+Payara IntelliJ plugin enables developers to easily configure Payara Server and 
+manage applications from the IntelliJ IDEA.
+
+* xref:documentation/ecosystem/intellij-plugin/payara-micro.adoc[Payara Micro] - 
+Payara IntelliJ plugin provides the support to create, run and debug Payara Micro 
+applications from the IntelliJ IDEA which is the microservices-ready version of the Payara Server.
```

Comment:
```suggestion
applications from IntelliJ IDEA. Payara Micro is the microservices-ready version of the Payara Server.
```
Sounded like you were saying Intellij IDEA is a microservices-ready version of Payara Server